### DISCUSSION TITLE
ASAM Test Specification Study Group

### DISCUSSION THREAD

**Discussion by jdsika** [openMSL/.github#4] [General]
Dear community,

I just stumpled upon a blog from Peak Solutions about the ongoing discussions within the ASAM Test Specification Study group. [Read it here](https://automotivetestingblog.wordpress.com/2023/02/06/an-asam-ods-model-for-test-data-management-in-the-field-of-adas-ad/)

We should consider looking into this when we propose a test infrastructure e.g. for perception sensor models @ClemensLinnhoff 

Best regards
Carlo

**Comment by ClemensLinnhoff** [comment]
This is very interesting! As far as I can see it has all the components of the testing we are envisioning  for OpenMSL. We should harmonize this with the definitions in the CSP and in the [Credibility Assessment Framework](https://github.com/virtual-vehicle/Credibility-Assessment-Framework). Then we would provide an application example for all these proposed processes.
